theseus_gui/src/pages/Settings.vue

```diff
@@ -1,7 +1,286 @@
+          <span class="slider">
+            Maximum Memory
+            <Slider v-model="javaMemory" :min="1024" :max="8192" :step="1024" />
+          </span>
+        </div>
+      </div>
+    </Card>
+    <Card class="settings-card">
+      <h2 class="settings-title">Window</h2>
+      <div class="settings-group">
+        <div class="settings-group">
+          <div class="sliders">
+            <span class="slider">
+              Width
+              <Slider v-model="javaMemory" :min="1024" :max="8192" :step="1024" />
+            </span>
+            <span class="slider">
+              Height
+              <Slider v-model="javaMemory" :min="1024" :max="8192" :step="1024" />
+            </span>
+          </div>
+          <div class="toggle-setting">
+            Start in Fullscreen
+            <input
+              id="fullscreen"
+              v-model="fullscreen"
+              type="checkbox"
+              name="fullscreen"
+              class="switch stylized-toggle"
+            />
+          </div>
+        </div>
+        <hr class="card-divider" />
+        <div class="settings-group">
+          <h3>Console</h3>
+          <div class="toggle-setting">
+            Show console while game is running
+            <input
+              id="fullscreen"
+              v-model="fullscreen"
+              type="checkbox"
+              name="fullscreen"
+              class="switch stylized-toggle"
+            />
+          </div>
+          <div class="toggle-setting">
+            Close console when game quits
+            <input
+              id="fullscreen"
+              v-model="fullscreen"
+              type="checkbox"
+              name="fullscreen"
+              class="switch stylized-toggle"
+            />
+          </div>
+          <div class="toggle-setting">
+            Show console when game crashes
+            <input
+              id="fullscreen"
+              v-model="fullscreen"
+              type="checkbox"
+              name="fullscreen"
+              class="switch stylized-toggle"
+            />
+          </div>
+        </div>
+      </div>
+    </Card>
+    <Card class="settings-card">
+      <h2 class="settings-title">Commands</h2>
+      <div class="settings-group">
+        <div class="toggle-setting">
+          Pre Launch
+          <input v-model="javaArgs" type="text" class="input" />
+        </div>
+        <div class="toggle-setting">
+          Wrapper
+          <input v-model="javaArgs" type="text" class="input" />
+        </div>
+        <div class="toggle-setting">
+          Post Launch
+          <input v-model="javaArgs" type="text" class="input" />
+        </div>
+      </div>
+    </Card>
   </div>
 </template>
+
+<style lang="scss">
+.slider-input {
+  width: 4rem !important;
+  flex-basis: 5rem !important;
+}
+
+.installation-input {
+  width: 100%;
+}
+
+.theming,
+.settings-card {
+  width: 90%;
+  margin: 2rem auto;
+  display: flex;
+  flex-direction: column;
```

Comment:
This style looks weird. I think just removing the `width` attr and making it just `margin: 1rem` would make it look a lot better